lib/git-hooks.js

```diff
@@ -111,6 +117,22 @@ module.exports = {
                 return path.resolve(hooksDirname, hookName);
             });
 
+            var ignoreList = [];
+            try {
```

Comment:
Just check `.gitignore` existence before running command and you don't need `try..catch` at all.